pt-BR/mail/chrome/messenger/accountCreation.dtd

```diff
@@ -57,30 +57,30 @@
-
-<!ENTITY insecureServer.tooltip.title    "Warning! This is an insecure server.">
-<!ENTITY insecureServer.tooltip.details  "Click circle for more details.">
-
-<!ENTITY insecureUnencrypted.description "Your email and authentication are sent unencrypted, so your password (and your message) could easily be read by other people. &brandShortName; will let you get to your mail, but you should contact your email provider about configuring the server with a secure connection.">
-<!ENTITY insecureSelfSigned.description  "The server uses a certificate that we can't trust, so we can't be sure that someone isn't intercepting the traffic between &brandShortName; and your server. &brandShortName; will let you get to your mail, but you should really get your email provider to configure the server with a trusted certificate.">
-<!ENTITY secureServer.description        "Congratulations! This is a secure server.">
-
-<!ENTITY pb.identity.title "Identity">
-<!ENTITY pb.name.label    "Your Name">
-<!ENTITY pb.email.label   "Email Address">
-<!ENTITY pb.password.label "Password">
-<!ENTITY pb.serverinfo.title "Server Information">
-<!ENTITY pb.username.label   "User Name">
-<!ENTITY pb.incoming.label "Incoming Server">
-<!ENTITY pb.servername.label "Server Name">
-<!ENTITY pb.servertype.label "Type">
-<!ENTITY pb.serverport.label "Port">
-<!ENTITY pb.serversecurity.label "Security">
-<!ENTITY pb.outgoing.label "Outgoing Server">
+<!ENTITY pb.contactYourProvider.description "O &brandShortName; pode permitir que você obtenha seus e-mails utilizando as configurações fornecidas. Entretanto, você deve contatar seu administrador ou provedor de e-mail em relação a essas conexões inadequadas.">
+
+<!ENTITY insecureServer.tooltip.title    "Atenção! Esse servidor é inseguro.">
+<!ENTITY insecureServer.tooltip.details  "Clique para mais detalhes.">
+
+<!ENTITY insecureUnencrypted.description "O e-mail e a autenticação são enviados sem criptografia, então sua senha (e sua mensagem) poderiam ser facilmente lidas por outras pessoas. O &brandShortName; permitirá que você receba seus e-mails, mas você deveria entrar em contato com o seu provedor para configurar o servidor usando uma conexão segura.">
+<!ENTITY insecureSelfSigned.description  "O servidor utiliza um certificado em que não podemos confiar. Portanto, não podemos ter certeza de que alguém não esteja interceptando o tráfego entre o &brandShortName; e o seu servidor. O &brandShortName; permitirá que você receba seus e-mails, mas você deveria pedir ao seu provedor que configure o servidor com um certificado confiável.">
+<!ENTITY secureServer.description        "Parabéns! Esse servidor é seguro.">
+
+<!ENTITY pb.identity.title "Identidade">
+<!ENTITY pb.name.label    "Seu nome">
+<!ENTITY pb.email.label   "Endereço de e-mail">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pb.email.label   "Endereço de correio eletrônico">